app/controllers/referee_interface/reference_controller.rb

```diff
@@ -4,7 +4,9 @@ class ReferenceController < ActionController::Base
     before_action :add_identity_to_log
     before_action :check_referee_has_valid_token
     before_action :set_token_param
-    before_action :show_finished_page_if_feedback_provided, except: %i[confirmation confirm_consent]
+    before_action :show_finished_page_if_feedback_provided, except: %i[submit_questionnaire submit_feedback confirmation]
```

Comment:
`only:` might be safer here as we don't need to remember to rule out any future actions we might happen to add

Reply:
🤔 I would have thought that the default behaviour post submitting their reference, is that they are redirected to the finish page. The submit_feedback page should not be there though.